.travis/before_install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+set -e
+
+cd $HOME
+
+if [[ $TRAVIS_PYTHON_VERSION == 'pypy' ]]; then
+    deactivate
+
+    wget https://bitbucket.org/squeaky/portable-pypy/downloads/pypy-5.8-linux_x86_64-portable.tar.bz2
```

Comment:
There are an awful lot of hardcoded paths in here, which might make this hard to update when 5.9 etc. come out? The version of this code I linked from the issue is more generic.